includes/class-boldgrid-backup.php

```diff
@@ -304,6 +304,8 @@ private function load_dependencies() {
 		require_once BOLDGRID_BACKUP_PATH . '/admin/orphan/class-finder.php';
 		require_once BOLDGRID_BACKUP_PATH . '/admin/orphan/class-cleanup.php';
 
+		require_once BOLDGRID_BACKUP_PATH . '/admin/class-boldgrid-backup-admin-environment.php';
```

Comment:
Sorry for not mentioning this in the last review, but I feel like we should have a comment here for what this require is for. We seem to have this for others such as WP-CLI Support. Logs System. Orphaned Files. I think that without this, we may think in the future that this require is for Orphaned files, not for a separate feature / functionality.

Reply:
Generally, I only added comments when there were groups of files included together that belonged to the same system. If it's just a one off file / class, that didn't have supplemental classes, I would just put it on a line all by itself.

If you scroll up higher in the file, you'll see the same format.

Reply:
A comment sounds good.  I started creating new classes using a folder hierarchy and namespaces.  This new file is not namespace and has a long filename.  Eventually, I was planning to convert all of the PHP files to use namespacing and organize all classes in folders, but that got backlogged.